packages/gatsby-cli/package.json

```diff
@@ -48,16 +45,29 @@
     "update-notifier": "^4.1.1",
     "uuid": "3.4.0",
     "yargs": "^15.4.1",
+    "yoga-layout-prebuilt": "^1.9.6",
```

Comment:
`yoga-layout-prebuilt` is dependency of `ink` but I had runtime problems when it was bundled. It does use `nbind` so I was thinking that's the issue with some form of binaries (at least with current rollup config, it might be possible to fully bundle it, but I didn't explore it much).

So because of bundling problems I had to mark it as external and make sure that users DO install it (so that's reason of why this is added do `dependencies` of `gatsby-cli`)

Reply:
https://github.com/gatsbyjs/gatsby/commit/c6c2368e02961f5c0d02213ffcaddb237c50b053 heh